packages/forms-web-app/src/views/before-you-start/local-planning-department.njk

```diff
@@ -0,0 +1,76 @@
+{% extends "layouts/main.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %},
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% set title="Which local planning department dealt with your planning application? - Before you start - Appeal a householder planning decision - GOV.UK" %}
+
+{% block pageTitle %}
+  {% if errors %}
+    {% set title="Error: " + title %}
+  {% endif %}
+  {{ title }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errorSummary %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-visually-hidden" data-cy="eligible-departments">{{ eligibleDepartments }}</p>
+      <p class="govuk-visually-hidden" data-cy="ineligible-departments">{{ ineligibleDepartments }}</p>
+
+      <form action="" method="POST" novalidate>
+        <div class="govuk-form-group {{ 'govuk-form-group--error' if errors }}">
+          <span class="govuk-caption-l">Before you start</span>
+
+          {{ govukSelect({
+            id: "local-planning-department",
+            name: "local-planning-department",
+            attributes: {"data-cy": "local-planning-department"},
+            label: {
+              text: "Which local planning department dealt with your planning application?",
+              classes: "govuk-label--l",
+              isPageHeading: true
+            },
+            hint: {
+              text:"This is the same thing as your 'local planning authority' or 'LPA'."
+            },
+            errorMessage: errors['local-planning-department'] and {
+              text: errors['local-planning-department'].msg
+            },
+            items: departments
+          }) }}
+
+        </div>
+
+        {{ govukButton({
+          text: "Continue",
+          type: "Submit",
```

Comment:
Submit should be in all lowercase (accessibility check) 
```suggestion
          type: "submit",
```

Reply:
Edited.